src/store/index.ts

```diff
@@ -1,11 +1,47 @@
 import Vue from "vue";
 import Vuex from "vuex";
+// import Language from "@/store/modules/Language";
+import VuexPersistence from "vuex-persist";
 
 Vue.use(Vuex);
 
-export default new Vuex.Store({
-  state: {},
-  mutations: {},
-  actions: {},
-  modules: {}
+// const debug = process.env.NODE_ENV !== "production";
+
+const vuexLocal = new VuexPersistence({
+  storage: window.localStorage
 });
+
+export default new Vuex.Store({
+  plugins: [vuexLocal.plugin]
+  // strict: debug,
+}); // import Vue from "vue";
```

Comment:
do we need these comments?

Reply:
In strict mode, whenever Vuex state is mutated outside of mutation handlers, an error will be thrown. This ensures that all state mutations can be explicitly tracked by debugging tools.